_site/posts/posts.json

```diff
@@ -1,4 +1,21 @@
+    "preview": {},
```

Comment:
the Positron extension looks really nice! 

Could you explain whether you'd consider making it behave like the Air one, at saving time?